test/build-wat.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
```

Comment:
Should this file be just be called `build.sh` or `build-wasm.sh`  (build-wat reads like the output of the script, would be wat, but its actually the input.. maybe `campile-wat.sh` if you want to keep wat in the name?)

Reply:
there is already `build.sh`, which build wasm from c.
i meant `build from wat`.

Reply:
Ah I see.  In that case perhaps we can replace the two different shell scripts with a single Makefile that by default converts all `.c` and `.wat` files?

No need to block this PR on that, but it might be nicer that creating a bunch of different "make" scripts.

Reply:
yes, i agree it's nice to have a unformed way to build tests.
at least it would help https://github.com/WebAssembly/wasi-testsuite/pull/41 .

Reply:
> or merge those two build scripts into a single `build.sh`

eventually, maybe.

at this moment C tests and WAT tests are worked on by different people.
merging them now just increases chances to conflict with eg. https://github.com/WebAssembly/wasi-threads/pull/25